AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -112,6 +112,17 @@ class TransactionConfigurator {
         }
     }
 
+    func estimateGasPrice() {
```

Comment:
Is `transaction.gasPrice` ever calculated earlier? Because `estimateGasPrice()` will override it.

Reply:
only for non dapp transactions